src/pages/Main/Carousel/Styled.Carousel.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+const S = {};
+
+S.Container = styled.div`
+  width: 1025px;
+  height: 300px;
+  position: relative;
+`;
+S.CarouselBox = styled.div`
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  display: flex;
+`;
```

Comment:
```suggestion
export const Container = styled.div`
  width: 1025px;
  height: 300px;
  position: relative;
`;

export const CarouselBox = styled.div`
  width: 100%;
  height: 100%;
  overflow: hidden;
  display: flex;
`;
```
관심사의 분리에 따른 아주 좋은 접근입니다! 다만 조금더 가독성이 좋은 방법을 알려드릴테니 참고하세요!
수정함에 따라 import 방식도 변경될텐데, 그 부분도 리뷰로 남겨두었으니 참고해주세요!